examples/hello-world/hello-pt/README.md

```diff
@@ -21,93 +28,16 @@ pip3 install -r requirements.txt
 
 ### 2. Run the experiment
 
-Prepare the data first:
-
-```
-bash ./prepare_data.sh
-```
-
-Use nvflare simulator to run the hello-examples:
+Run the script using the job API to create the job and run it with the simulator:
 
 ```
-nvflare simulator -w /tmp/nvflare/ -n 2 -t 2 hello-pt/jobs/hello-pt
+python3 fedavg_script_executor_hello-pt.py
 ```
 
 ### 3. Access the logs and results
 
-You can find the running logs and results inside the simulator's workspace/simulate_job
+You can find the running logs and results inside the simulator's workspace:
 
 ```bash
-$ ls /tmp/nvflare/simulate_job/
-app_server  app_site-1  app_site-2  log.txt
-
-```
-
-### 4. Global Model Initialization Approaches
```

Comment:
We maybe need another example to explain this, don't need to be in hello example though

Reply:
Yes, this relates to the comment you made on hello-numpy initial params from client side, I did try but had issues with Job API to do it with the existing components, so this should be showcased in another example. After this PR in a new one perhaps.